toqito/state_props/entanglement_of_formation.py

```diff
@@ -40,16 +40,8 @@ def entanglement_of_formation(rho: np.ndarray, dim: list[int] | int = None) -> f
     >>>
     >>> u_vec = bell(0)
     >>> rho = u_vec * u_vec.conj().T
-    >>> '%.2f' % entanglement_of_formation(rho)
-    '1.00'
-
-    .. note::
-        You do not need to use `'%.2f' %` when you use this function.
-
-        We use this to format our output such that `doctest` compares the calculated output to the
-        expected output upto two decimal points only. The accuracy of the solvers can calculate the
-        `float` output to a certain amount of precision such that the value deviates after a few digits
-        of accuracy.
+    >>> np.around(entanglement_of_formation(rho), decimals=2)
+    1.00
```

Comment:
```suggestion
    >>> import numpy as np
    >>> np.around(entanglement_of_formation(rho), decimals=2)
    1.00
```
Sometimes `np.around` will round up to a higher number when we don't want it to. We don't see that here but when a function relies on optimization, `np.around(0.8)` gets rounded up to `1.0`. This is why I did not use it.

Reply:
Yes, but this would only occur if you don't specify the number of decimals. 

So, for instance:

```py
np.around(0.66)
1.0
```

However

```py
np.around(0.66, decimals=2)
0.66
```